app/lib/ctc_signup_message.rb

```diff
@@ -38,4 +38,37 @@ def self.send(count)
       ctc_signup.touch(:beta_email_sent_at)
     end
   end
-end
\ No newline at end of file
+
+  def self.send_launch_announcement(count)
```

Comment:
the request for the beta emails was to send in chunks with specific counts, but here i think the request is to send to everyone on the list. what if passing nothing for the count got all of them?

Reply:
I was thinking I'd pass in `10` first in production, make sure nothing blew up, then do them in batches of `100` or `1000` to make sure nothing blew up as a way to do all of them. Treating a missing `count` as infinity is fine too, shrug.

FWIW, I was also thinking that I'd use `def self._send_one_launch_announcement` to rehearse with a Kelly sign-up in production.